tests/test_simulated_annealing.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from sklearn.datasets import make_friedman1
+from sklearn.linear_model import LinearRegression
+
+from feature_selection import simulated_annealing
+
+def scorer(X, y):
```

Comment:
I believe that in some part of the documentation it should be mention that the scorer should return a results where "the smaller, the better"

Reply:
Agreed, flagged for Milestone 3.